ncm-network/src/main/perl/network.pm

```diff
@@ -839,6 +846,14 @@ sub process_network
             }
             $iface->{my_inner_ipaddr} .= "/$iface->{my_inner_prefix}";
         }
+
+        # Newly added used when generating the keyfile format config in networkmanager module
+        my $media =  $nic->{media};
+        if ($media) {
+            $iface->{media} = $media;
```

Comment:
is this something we should support in the schema? if you want to set (or is it preserve?) it, you should do this via the schema.